clang/include/clang/AST/ExprCXX.h

```diff
@@ -4377,15 +4377,20 @@ class PackIndexingExpr final
   // The pack being indexed, followed by the index
   Stmt *SubExprs[2];
 
-  size_t TransformedExpressions;
+  // The size of the trailing expressions.
+  unsigned TransformedExpressions : 31;
```

Comment:
I changed it to `unsigned` because `numTrailingObjects` is designed to return an `unsigned` number.

Reply:
Aside: I'm not spelling it as `: sizeof(unsigned) - 1` because we seem to write down the number 31 directly elsewhere. So I think i'd better respect the convention.